### PR TITLE
docs-14: Improve execution methods section

### PR DESCRIPTION
**Pull request by olavandeira**
- Added details on how to use the web client
- Added Postman execution method

### CODE REVIEW

File: api-reference/quickstart.mdx [hunk 1]
```diff
@@ -63,12 +63,29 @@ Our API documentation supports various testing and execution methods to suit dif
...
+- **Try it out**: Import our API collection to Postman for a seamless transition from our documentation to your development environment.
+
+<a href="https://god.gw.postman.com/run-collection/34096189-b77feaba-4ca9-4456-a365-6a771e02ccad?action=collection%2Ffork&source=rip_markdown&collection-url=entityId%3D34096189-b77feaba-4ca9-4456-a365-6a771e02ccad%26entityType%3Dcollection%26workspaceId%3Dfcdc3a3c-e3c2-476d-9607-b265b44de679#?env%5BSandbox%5D=W3sia2V5IjoiYmFzZV91cmwiLCJ2YWx1ZSI6ImFwaS5zYW5kYm94LmdvdGVhbC5jbyIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJkZWZhdWx0Iiwic2Vzc2lvblZhbHVlIjoiYXBpLnNhbmRib3guZ290ZWFsLmNvIiwic2Vzc2lvbkluZGV4IjowfSx7ImtleSI6InNjaGVtZSIsInZhbHVlIjoiaHR0cHMiLCJlbmFibGVkIjp0cnVlLCJ0eXBlIjoiZGVmYXVsdCIsInNlc3Npb25WYWx1ZSI6Imh0dHBzIiwic2Vzc2lvbkluZGV4IjoxfSx7ImtleSI6InVzZXJfaWQiLCJ2YWx1ZSI6IiIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJhbnkiLCJzZXNzaW9uVmFsdWUiOiJudWxsIiwic2Vzc2lvbkluZGV4IjoyfV0=" target="_blank" rel="noopener">
+    <div style={{ height:'32px', width: '128px', margin:'32px 0px', backgroundImage: 'url("https://run.pstmn.io/button.svg")', backgroundSize: 'cover', backgroundPosition: 'center' }}></div>
+</a>
```
Comment (luchesar) [inline api-reference/quickstart.mdx:81]: Looks much better this way!
Would it be good to add some screenshots about the Profile variables and where to look for them in Postman?
It is quite confusing in the begining where those variables are and what their values are.

**Review by luchesar — APPROVED** [review]
Just one comment, but I think it's worth doing it.

File: api-reference/quickstart.mdx (hunk 1 above)
Comment (olavandeira) [reply api-reference/quickstart.mdx:81]: Updated

I understand that you are referring to the environment variables

**Comment by olavandeira** [comment]
@luchesar Updated based on feedback and ready to review

**Review by arun-singh — APPROVED** [review]

**Pull request merged**